tachyon/zk/lookup/verifier.h

```diff
@@ -4,7 +4,6 @@
 #include <memory>
 #include <vector>
 
-#include "tachyon/zk/lookup/type.h"
```

Comment:
I think for this change, it just should not have been included from the previous commit.